docs/tutorials/how-to-create-template.md

```diff
@@ -14,11 +14,11 @@ This tutorial explains how to create a Alibaba Cloud EC2 template and use it to
 ## Create a repository on GitHub
 
 1. Create a new repository on GitHub of your own.
- Here we use the repository [demo](https://github.com/walrus-catalog-demo/demo)
+ Here we use the repository [terraform-alicloud-rds-mysql](https://github.com/walrus-catalog/terraform-alicloud-rds-mysql)
```

Comment:
There's no demo directory in the repo. 
Why users need to create files if all the files are present after the clone?

Reply:
This is an example to show files should be created for a new template if users want to create their own template.

We may add introduction here, like `Your can replace the git with your own and create the following files or using our repo to modify as your template`?

Reply:
Updated, using a simple kubernetes template to help user quickly create their template.